lib/l10n.js

```diff
@@ -13,456 +13,569 @@ var l10n = {
     },
     msg_unsubscribed_default: {
-        en: 'You are unsubscribed from Fight for the Future. No more messages will be sent. Email team@hello.vote for help or to restart.',
+        en: 'You are unsubscribed from Fight for the Future. No more messages will be sent. Email team@hello.vote for help or to restart.',,
+        es: "Te desinscribes de Lucha por el futuro. No se enviarán más mensajes. Envíe un correo electrónico a team@hello.vote para obtener ayuda o para reiniciar.",
+        ko: "미래를 쟁취하자 재단의 이메일 소시지 구독을 해지하셨습니다. 더 이상 이메일을 받지 않으실 겁니다. 도움을 요청하거나 이메일을 다시 받기 위해서는 team@hello.vote 에 이메일을 보내주세요."
     },
     msg_help: {
         en: 'To go back a step, say BACK. To unsubscribe, say STOP. To start over, re-enter your # on hellovote.org. All other questions, email team@hello.vote',
-        es: "Para regresar, diga REGRESAR. Para anular su suscripción, diga STOP. Para volver a empezar, re-ingrese su # en holavote.org. Para otras preguntas envíe un mensaje a team@hello.vote"
+        es: "Para regresar, diga REGRESAR. Para anular su suscripción, diga STOP. Para volver a empezar, re-ingrese su # en holavote.org. Para otras preguntas envíe un mensaje a team@hello.vote",
+        ko: "전 단계로 돌아가기 위해서는 back 을 입력하세요. 구독을 해지하기 위해서는 stop 을 입력하세요. 처음부터 시작하기 위해서는 hellovote.org 를 방문해서 전화번호를 입력하세요. 질문은 team@hello.vote 으로 보내주세요."
     },
     msg_help_default: {
         en: 'Fight for the Future. Help at team@hello.vote or 855-447-3383. Msg&data rates may apply. 4 msgs/month. To unsubscribe, text STOP',
+        es: '“Fight for the Future.” Para ayuda - team@hello.vote o 855-447-3383. Datos pueden ser aplicados para desconectar mande mensaje diciendo “stop”',
+        ko: "활동에 동참 해 주세요! 이메일 team@hello.vote 또는 855-447-3383 으로 연락 주세요.메세지 또는 데이터 요금이 적용 될 수 있습니다. 한달에 4 메세지. 구독 해지를 하기 원하서는 STOP 이라고 입력하세요."
     },
     msg_try_again: {
         en: 'Please try again!',
-        es: "¡Por favor pruebe otra vez!"
+        es: "¡Por favor pruebe otra vez!",
+        ko: "다시 시도 해 주세요."
     },
     msg_error_unknown: {
         en: 'I seem to have had a glitch. Please send your last message again.',
-        es: "Ocurrió una falla. Por favor envíe su último mensaje otra vez."
+        es: "Ocurrió una falla. Por favor envíe su último mensaje otra vez.",
+        ko: "무언가 문제가 생겼습니다. 마지막 메세지를 다시 보내주세요."
     },
     msg_error_failed: {
         en: 'Agh, an error occurred and your voter registration could not be processed. Please let us know: team@hellovote.org - reference #',
-        es: "¡Ay! Un error occurio y su inscripción para votar no pudo ser procesada. Por favor avísenos: team@hellovote.org - # de referencia  "
+        es: "¡Ay! Un error occurio y su inscripción para votar no pudo ser procesada. Por favor avísenos: team@hellovote.org - # de referencia  ",
+        ko: "유권자 등록 처리에 문제가 생겼습니다. 죄송합니다. 레퍼런스 번호 #{{reference}} 를 이메일 team@hellovote.org 으로 보내서 알려주세요"
```

Comment:
We append the reference to this message, it is not templated in.

Reply:
will ask for a retranslation with the reference number at the end